azure-pipelines.yml

```diff
@@ -30,6 +33,15 @@ stages:
     steps:
     - template: FhirToDataLake/build.yml
 
+- stage: e2e
+  pool:
+    vmImage: 'windows-latest'
+  condition: succeeded()
+  jobs:
+  - job: SynapseE2ETest
```

Comment:
nit, maybe we can rename to `RunSynapseE2ETest` as all job names in this file are in verb+noun form.

Reply:
Updated, thanks.